### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by jmikola**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,4 +59,219 @@ These are automatically executed by the `make distcheck`, which will package a n
...
+
+Ensure that the generated PECL package can be successfully installed:
+
+```
+$ pecl install -f mongodb-X.Y.Z.tgz
+```
+
+The PECL package may be published via the
+[Release Upload](https://pecl.php.net/release-upload.php) form. You will have
+one change to confirm the package information after uploading.
```
Comment (bjori) [inline CONTRIBUTING.md]: one `chance`?

Comment (jmikola) [inline CONTRIBUTING.md]: Yup :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -59,4 +59,219 @@ These are automatically executed by the `make distcheck`, which will package a n
...
+The build process will resemble:
+
+```
+cd c:\php-sdk\
+bin\phpsdk_setvars.bat
+
+cd C:\php-sdk\phpdev\vc11\x86\php-5.6.12-src
+nmake clean
+buildconf --force
+configure --disable-all --with-openssl --enable-cli --enable-json --enable-mongodb
```
Comment (derickr) [inline CONTRIBUTING.md]: It should really test whether it compiles shared, and not just built-in.

Comment (jmikola) [inline CONTRIBUTING.md]: If you have instructions for doing so, I'd be happy to add them. It isn't covered in the wiki docs, AFAIK.

**Comment by jmikola** [comment]
Merging this as-is. If you have additional input, feel free to correct it in a subsequent PR.

**Pull request merged**